api/src/chat/services/block.service.ts

```
@@ -46,15 +59,40 @@ export class BlockService extends BaseService<
+    @Optional() gateway?: WebsocketGateway,
   ) {
     super(repository);
+    if (gateway) {
+      this.gateway = gateway;
+    }
+  }
+
+  @SocketGet('/block/subscribe/')
+  @SocketPost('/block/subscribe/')
+  subscribe(@SocketReq() req: SocketRequest, @SocketRes() res: SocketResponse) {
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)